### ISSUE TITLE
Updates to listing notification email 

### ISSUE THREAD

**Issue by sarahlazarich** [metrotranscom/doorway#995]
Doorway would like to add some language to the listing notification email 
- Add listing due time to new listing notification emails
- Add a sentence  in the email that says “Please refer to the listing for the most accurate information, as it is subject to change.”

AC: 
- The listing closure time is displayed on the email (in addition to the date) 
- Date and times on email blast are displayed in the email in PST 
- New copy is displayed on the listing

**Comment by sarahlazarich** [comment]
Placement of new line of copy TBD and needs to be approved by BAHFA 
We have a template to work off of

**Comment by sarahlazarich** [comment]
Hard to test without access to govdelivery 
Morgan will have to set that up for whoever is testing it, account is owned by BAHFA 
Morgan can't add everyone to the UI but can add everyone to the distribution list

**Comment by jaredcwhite** [comment]
Looks like we don't have a unit test for this email, so I'm adding one to aid development as well as validate in automated tests.

**Comment by ludtkemorgan** [comment]
@sarahlazarich good catch. The lottery date/time is saved in two separate fields and are only looking at the date field. I can update it as part of my bulk email change PR.

**Comment by jaredcwhite** [comment]
Ohh, dang. Sorry I missed that!

**Comment by sarahlazarich** [comment]
small thing @jaredcwhite and @ludtkemorgan - can we add a space into that lottery field between the year and "at" 

<img width="379" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBdzVuQ0E9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4a743cd17424a463f74ae530c28ac37936337164/image.png" alt="image.png" />

**Comment by ludtkemorgan** [comment]
@sarahlazarich good catch. I made a fix for this in a new PR for bash findings and will go in before the release. https://github.com/metrotranscom/doorway/pull/1043
